### PR TITLE
fix some modal styles and loader styles

### PR DESCRIPTION
**Pull request by ArturBekhDEV**
fix some email modal styles  and loader styles in categories and subject page.

### CODE REVIEW

**Review by FryDay969 — APPROVED** [review]

File: src/components/cards-list/CardsList.styles.ts [hunk 1]
```diff
@@ -14,5 +14,9 @@ export const styles = {
     minWidth: '148px',
     display: 'block',
     m: '32px auto 0'
+  },
+  loaderContainer: {
+    height: '50vh',
```
Comment (MickhaelSh) [inline src/components/cards-list/CardsList.styles.ts:19]: why we need 50vh here?

File: src/containers/email-confirm-modal/EmailConfirmModal.styles.js [hunk 2]
```diff
@@ -6,14 +6,16 @@ export const styles = {
...
-    color: 'palette.grey.500'
-  },
   button: {
     size: 'large',
     mt: '32px'
+  },
+  titleWithDescription: {
+    title: {
+      typography: 'h6',
+      color: 'palette.grey.500',
```
Comment (MickhaelSh) [inline src/containers/email-confirm-modal/EmailConfirmModal.styles.js:16]: primary.500

File: src/components/cards-list/CardsList.styles.ts (hunk 1 above)
Comment (ArturBekhDEV) [reply src/components/cards-list/CardsList.styles.ts]: changed to minHeight

File: src/containers/email-confirm-modal/EmailConfirmModal.styles.js (hunk 2 above)
Comment (ArturBekhDEV) [reply src/containers/email-confirm-modal/EmailConfirmModal.styles.js]: changed to primary.900, relying on figma styles

File: src/containers/email-confirm-modal/EmailConfirmModal.styles.js [hunk 3]
```diff
@@ -6,14 +6,16 @@ export const styles = {
...
-  },
   button: {
     size: 'large',
     mt: '32px'
+  },
+  titleWithDescription: {
+    title: {
+      typography: 'h6',
+      color: 'primary.900',
+      fontWeight: '600',
```
Comment (MickhaelSh) [inline src/containers/email-confirm-modal/EmailConfirmModal.styles.js:17]: on figma 500, dellete this, h6 alredy has 500

**Review by maarekua — APPROVED** [review]

File: src/containers/email-confirm-modal/EmailConfirmModal.styles.js (hunk 3 above)
Comment (ArturBekhDEV) [reply src/containers/email-confirm-modal/EmailConfirmModal.styles.js]: +

**Review by maarekua — APPROVED** [review]

**Pull request merged**